easybuild/easyconfigs/p/picard/picard-2.18.14.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::	MIT 
+#
+# 2.10.1:
+# Adam Huffman
+# The Francis Crick Institute
+# 2.18.11:
+# Jonas Demeulemeester
+# The Francis Crick Institute
+##
+
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.18.14'
```

Comment:
@zao Any reason you dropped the traditional versionsuffix (also requires renaming the easyconfig)?
```python
versionsuffix = '-Java-%(javaver)s'
```

Reply:
I believe I was inspired by the older versions that didn't have a versionsuffix despite having a Java dependency inside.
Addressed this by renaming the easyconfig and adding a versionsuffix.